frontend/src/store.js

```diff
@@ -32,7 +32,7 @@ export default new Vuex.Store({
       commit(MUTATIONS.SET_STUDENTS, data)
     },
     ...nativeLanguageActions,
-    ...screeningActions
+    ...screeningActions,
```

Comment:
`nativeLanguageActions` also appears twice, once on Line 27 and once on Line 34. Let's remove one, and keep all the spread actions together:

```js
actions: {
  ...someActions,
  ...otherActions,
  ...evenMoreActions,
  async getAllStudents() { ... },
  async anotherAction() { ... }
}
```

Reply:
okie, made changes in a new commit